index.html

```diff
@@ -17,12 +18,71 @@
   <body>
 
     <header>
+<<<<<<< HEAD
+      <h1 class="title">Fun at the Zoo!</h1>
+=======
       <h1 class="title">Mad-Libs</h1>
       <p class="instructions">Fill in the blanks!</p>
+>>>>>>> f4a133e526a33cdb195bc25c23e706da606e6ab7
     </header>
 
     <main>
+      <section class="instructions">
+        <div>How to Play:</div>
+        <div>Fill in the blanks!</div>
+        <div>Then click the button!</div>
+      </section>
+
+<<<<<<< HEAD
+      <article class="zoo">
+        Today I went to the zoo. I saw
+        <span id="a-or-an">a(n)</span>
+        <span id="adj-1">(adjective)</span>
+        <span id="noun-1">(Noun)</span>
+        jumping up and down in its tree. He
+        <span id="verb-pt-1">(verb, past tense)</span>
+        <span id="adverb-1">(adverb)</span>
+        through the large tunnel that led to its
+        <span id="adj-2">(adjective)</span>
+        <span id="noun-2">(noun)</span>. I got some peanuts and passed them
+        through the cage to a gigantic gray
+        <span id="noun-3">(noun)</span>
+        towering above my head. Feeding that animal made me hungry. I went to
+        get a
+        <span id="adj-3">(adjective)</span>
+        scoop of ice cream. It filled my stomach. Afterwards I had to
+        <span id="verb-1">(verb)</span>
+        <span id="adverb-2">(adverb)</span>
+        to catch our bus. When I got home I
+        <span id="verb-pt-2">(verb, past tense)</span>
+        my mom for a
+        <span id="adj4">(adjective)</span> day at the zoo.
+      </article>
 
+      <section class="container">
```

Comment:
Either this container or the div below it needs the ID for the JavaScript to access it